perses/rjmc/geometry.py

```diff
@@ -117,16 +117,22 @@ def propose(self, top_proposal):
 
                 #propose a bond and calculate its probability
                 bond = self._get_relevant_bond(atom, bond_atom)
-                r_proposed = self._propose_bond(bond, top_proposal.beta)
-                logp_r = self._bond_logp(r_proposed, bond, top_proposal.beta)
+                r_proposed = self._propose_bond(bond, beta)
+                bond_k = bond.type.k*units.kilocalorie_per_mole/units.angstrom**2
+                sigma_r = units.sqrt(1/beta*bond_k)
+                logZ_r = np.log((np.sqrt(2*np.pi)*sigma_r/sigma_r.unit))
```

Comment:
Suggest you use a standard fixed unit (e.g. unit.angstrom) here just in case we end up changing the units of `sigma_r` or `beta` above.